plugins/inputs/cloudwatch/sample.conf

```
@@ -21,7 +21,12 @@
   # role_session_name = ""
   # profile = ""
   # shared_credential_file = ""
-
+  
+  ## If you are using CloudWatch cross-account observability, you can 
+  ## set IncludeLinkedAccounts to true in a monitoring account 
+  ## and collect metrics from the linked source accounts
+  # IncludeLinkedAccounts = false
+  
```

Please use snake_case for options! I think you also did for the TOML annotation in code, but did not yet for the documentation
```suggestion
  ## If you are using CloudWatch cross-account observability, you can 
  ## set IncludeLinkedAccounts to true in a monitoring account 
  ## and collect metrics from the linked source accounts
  # include_linked_accounts = false
  
```